doit/apps/tasks/tests.py

```diff
@@ -40,6 +40,32 @@ def test_list_task(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_create_task(self):
+        dummy_data = {
+            "title": "İlaç iç.",
+            "description": "Mide ilaçlarını iç.",
+            "reminders": [
+                {
+                    "date": "2017-08-15T12:59:44Z"
+                },
+                {
+                    "date": "2017-08-15T12:59:53Z"
+                }
+            ]
+        }
+        url = reverse('v1:tasks')
+
+        # Create Task
+        response = self.client.post(url, dummy_data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Check Task Info
```

Comment:
Bu alanı kaldıralım. Burada task için sorgu atarken user'e göre çekmen sorun olabilir.